src/debug.ts

```diff
@@ -0,0 +1,2 @@
+const debug = false
+export default debug
```

Comment:
It would be better to create a function for debug output in debug.ts.

Reply:
How should debugging be managed? As an option in `BrowserWindow` is quite poor because the debugging is global. Exporting debug could be an option.

Reply:
I think it would be better to create a function for the output for debug.

Reply:
Should the function toggle debugging or log the debug output?

Reply:
it should log the debug output.